README.adoc

```diff
@@ -97,6 +99,10 @@ mvn liberty:stop
 // =================================================================================================
 == Defining a property with expressions
 
+In MicroProfile Config, property expressions provide a way to set and expand variables in property values. To use property expressions, define a property with a `${expression}` syntax, where `expression` is the name of another configuration property. When the property is loaded, the `${expression}` syntax is replaced with the value of the referenced property.
```

Comment:
I think the value of property expressions is that you can make the value of one property depend on the value of another property- can you think of a concise example where this would be beneficial? perhaps when you need to specify a port value or similar? Doesnt need to be much but could go in between the first and second sentence in this paragraph.

Reply:
Edited in https://github.com/OpenLiberty/draft-guide-microprofile-config-apis/pull/17/commits/cdee87d7a502e51b1c3328cfe2e7972aaf4d5a3c